doc/api/stream.md

```diff
@@ -28,10 +28,13 @@ objects will rarely (if ever) have need to use the `stream` module directly.
 
 ## Organization of this Document
 
-This document is divided into two primary sections with a third section for
+This document is divided into four primary sections with a fifth section for
 additional notes. The first section explains the elements of the stream API that
 are required to *use* streams within an application. The second section explains
 the elements of the API that are required to *implement* new types of streams.
+The third section is geared towards explanations and *code examples*. The fourth
```

Comment:
Nit: "geared towards explanations" is less informative than this should be IMO. Maybe this?

```md
The third section explains <whatever it is supposed to explain> including code samples.
```

Or perhaps even better, this?

```md
The third section contains code samples.
```

Reply:
Sounds good! Will update